client/src/containers/widget/widget-header/index.tsx

```diff
@@ -23,7 +33,26 @@ const WidgetHeader: FC<WidgetHeaderProps> = ({
         <Title as="h3" className="text-base">
           {indicator}
         </Title>
-        {menu}
+        <div className="flex items-center gap-2">
```

Comment:
I would hide the response rate if there's no data in the widget. But this is a design call, so feel free to ignore it.